docs/apps/components/button.md

```diff
@@ -0,0 +1,83 @@
+
+3. **Perform Data Manipulation or Calculations**: It's useful for real-time validation, dynamic calculations, or updating form fields based on other inputs.
+
+4. **Navigate to Different Pages or Sections**: It allows for multi-step forms, progressive disclosure, or branching based on user choices.
+
+5. **Trigger Animations or Visual Effects**: Buttons can initiate animations or visual effects to enhance the user experience and provide visual feedback. It can include highlighting fields upon clicking, fading in or out elements, or displaying progress indicators.
+
+6. **Open pop-up windows or dialogs**: Buttons can be used to open pop-up windows or dialogs for displaying additional information, collecting supplementary data, or providing context-sensitive guidance.
+
+7. **Trigger custom actions or workflows**: It allows for extending the form builder's functionality and integrating it with external systems.
+
+## API
+
+### Events
+
+| **Name**| **Description**|
+|----------------------|---------------------------------------------------------------------|
+| **onClick (element)**| Event handler that's triggered when the button is clicked. This event handler can be used to perform any number of actions, such as submitting the form, redirecting the user to a different page, or displaying an error message|
+
+### Methods
+
+| **Name**| **Description**|
+|----------------------|---------------------------------------------------------------------|
+|**this.addClassName(fields, disabled)**|Add style class to form item|
+|**this.removeClassName(fields, disabled)**|Remove form item style|
+| **this.disable(fields)**| Disable input field from user interaction|
+| **this.enable(fields)**| Disable input field from user interaction|
+| **this.getComponent(name) → {Object}**|Retrieve a component from an object or element|
+| **this.getValue(fieldName)**|Get A Value From a component|
+|**this.hide(fields)**|Hides the field|
+|**this.show(fields)**|Displays the field|
+
+## Config
+
+| **Name**|**Description**|**Image**|
+|---------------|----------------------------------------------------------------------------------------------------------------------------------------|-----|
+|**ID**| This is a unique identifier which is used to access the field by the API and the key of the field when the form is saved|<img src= "/apps/components/img/alert_id2.png">|
+|**Name**| [**Optional**] An identifier for the button within the form. It's used by the form builder to associate the button with the data it's submitting. The name is typically not displayed to the user|<img src= "/apps/components/img/button_name.png">|
+|**Button Name**| [**Optional**] The visual identifier label of the field|<img src= "/apps/components/img/button_buttonname.png">|
+|**Width**| [**Optional**] To set the width of the field|<img src= "/apps/components/img/input_width.png">|
```

Comment:
This can be shortened further to: **The width of the field.**. Saying to set sounds like an action instead of a property.

Reply:
done
The width of the field. Example: 20%, 20px, 20rem